src/Robo/Commands/Setup/ConfigCommand.php

```diff
@@ -66,6 +66,13 @@ public function import() {
             $task->exec("drush @$drush_alias pm-enable config_split --yes");
             $task->exec("drush @$drush_alias config-import sync --yes");
           }
+          if (!$this->getConfigValue('cm.allow-overrides')) {
+            $this->say("Checking for config overrides...");
+            $config_overrides = $task->exec("drush @$drush_alias cex sync -n | grep 'Differences of the active config'");
```

Comment:
I don't think task output is captured quite the same as Phing, this will need some work